.github/workflows/python-release.yml

```diff
@@ -0,0 +1,34 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install build
```

Comment:
I believe the build will only use the requirements stored in the setup.py file. But I'm not 100% sure on this since the publish job hasn't actually been tested 😅